source/hardware/controllers/stanton_djc_4.rst

```diff
@@ -232,3 +173,17 @@ autoShowFourDecks If a track gets loaded into deck 3 or 4, should automatically
 showMasterVu      If set to false, show channel VU meter instead of Master L/R                               true
 dryWetAdjustValue Amount the dryWetKnob changes the value for each increment                                 0.05
 ================= ========================================================================================== =======
+
+Known Issues
+^^^^^^^^^^^^
+
+The following controls not mapped in this mapping:
```

Comment:
The following controls **are** not mapped in this mapping

Reply:
I'm on my phone, can you make a suggestion? Then I can commit this right now ;-)

Reply:
Nevermind, I'll just grab my laptop.

Reply:
for some reason I could not make a suggestion...